docs/resources.md

```diff
@@ -84,8 +89,27 @@ Generic Attestation Policy Spec description:
 
 | Field     | Default (if applicable)   | Description |
 |-----------|---------------------------|-------------|
+| attestationAuthorityNames | | Non-empty List of [Attestation Authorities](#attestationauthority-crd) for which all of them are required to be satisfied before the Admission Controller will admit the pod.|
 | attestationAuthorityNames | | List of [Attestation Authorities](#attestationauthority-crd) for which one of is required to be satisfied before the Admission Controller will admit the pod.|
 
+Note that the list of [Attestation Authorities](#attestationauthority-crd) must be non-empty. If the list is empty, an error will be thrown for malformed policy, and no image will be admitted, including allowlisted images.
+
+Admission Allowlist Pattern Spec description
+
+| Field     | Default (if applicable)   | Description |
+|-----------|---------------------------|-------------|
+| namePattern | | A name pattern that specifies which images are not inspected by Admission Controller.|
```

Comment:
nit: perhaps rephrase "which images are not inspected by Admission Controler" to "which images are allowed to pass through"? The main difference being avoiding the use of negatives, to simplify it for the reader.

Reply:
done.